### PR TITLE
Added my profile

### PR DESCRIPTION
**Pull request by dhruvmendpara1**
Edited code to add myself to the website

### CODE REVIEW

File: 2-members.md [hunk 1]
```diff
@@ -311,6 +311,28 @@ banner_color: style2
...
+   <img src="assets/images/members/Dhruv Mendpara.jpeg" alt="" data-position="center center" />
+  </a>
+  <div class="content">
+   <div class="inner">
+    <header class="major">
+     <h3>Dhruv Mendpara</h3>
+    </header>
+    <li>Joined in Fall 2024.</li>
+    <li>Undergraduate, Chemical Engineering, Purdue University</li>
+    <p><b>Research topic</b>: Optimization of tools.</p>
```
Comment (bernalde) [inline 2-members.md:327]: Edit this to "Parameter setting for optimization solvers."

File: 2-members.md [hunk 2]
```diff
@@ -311,6 +311,28 @@ banner_color: style2
...
+  <a class="image">
+   <img src="assets/images/members/Dhruv Mendpara.jpeg" alt="" data-position="center center" />
+  </a>
+  <div class="content">
+   <div class="inner">
+    <header class="major">
+     <h3>Dhruv Mendpara</h3>
+    </header>
+    <li>Joined in Fall 2024.</li>
+    <li>Undergraduate, Chemical Engineering, Purdue University</li>
```
Comment (bernalde) [inline 2-members.md:326]: Missing graduation date. Follow example from Keegan

File: 2-members.md [hunk 3]
```diff
@@ -311,6 +311,28 @@ banner_color: style2
  </section>
 </section>
 
+<!-- 3.5 -->
+<section id="three-five" class="spotlights">
+ <section>
+  <a class="image">
+   <img src="assets/images/members/Dhruv Mendpara.jpeg" alt="" data-position="center center" />
```
Comment (bernalde) [inline 2-members.md:318]: Have you uploaded the picture?

**Review by bernalde — CHANGES_REQUESTED** [review]
Some changes need to be addressed before merging. See the comments

File: 2-members.md (hunk 3 above)
Comment (dhruvmendpara1) [reply 2-members.md:318]: I did upload it and made a pull request for it.

**Review by bernalde — APPROVED** [review]

**Pull request merged**